cmd/clients/main.go

```diff
@@ -313,7 +330,20 @@ func main() {
 	}
 }
 
-func newService(ctx context.Context, db *sqlx.DB, dbConfig pgclient.Config, authz smqauthz.Authorization, pe policies.Evaluator, ps policies.Service, cacheClient *redis.Client, cfg config, channels grpcChannelsV1.ChannelsServiceClient, groups grpcGroupsV1.GroupsServiceClient, tracer trace.Tracer, logger *slog.Logger) (clients.Service, pClients.Service, error) {
+func newService(ctx context.Context,
+	db *sqlx.DB,
+	dbConfig pgclient.Config,
+	authz smqauthz.Authorization,
+	pe policies.Evaluator,
+	ps policies.Service,
+	cacheClient *redis.Client,
+	cfg config,
+	channels grpcChannelsV1.ChannelsServiceClient,
+	groups grpcGroupsV1.GroupsServiceClient,
+	tracer trace.Tracer,
+	logger *slog.Logger,
+	callout callout.Callout,
+) (clients.Service, pClients.Service, error) {
```

Comment:
Can we make this single line  ?
Did linter throw errors if we make single line  ?